pyTigerGraph/pyTigerGraphAuth.py

```diff
@@ -226,33 +226,25 @@ def getToken(self, secret: str, setToken: bool = True, lifetime: int = None) ->
 
         if int(s) < 3 or (int(s) == 3 and int(m) < 5):
             try:
-                if self.useCert and self.certPath:
-                    res = json.loads(requests.request("GET", self.restppUrl +
-                        "/requesttoken?secret=" + secret +
-                        ("&lifetime=" + str(lifetime) if lifetime else "")).text)
-                else:
-                    res = json.loads(requests.request("GET", self.restppUrl +
-                        "/requesttoken?secret=" + secret +
-                        ("&lifetime=" + str(lifetime) if lifetime else ""), verify=False).text)
+                res = json.loads(requests.request("GET", self.restppUrl +
+                    "/requesttoken?secret=" + secret +
+                    ("&lifetime=" + str(lifetime) if lifetime else ""), verify=False).text)
                 if not res["error"]:
                     success = True
-            except:
-                success = False
+            except Exception as e:
```

Comment:
just nitpicking: if catch and raise without doing something else, why not remove try...except altogether?